ui/app/components/job-status/panel/deploying.hbs

```diff
@@ -27,18 +27,36 @@
               @disabled={{this.fail.isRunning}}
               @onConfirm={{perform this.fail}} />
           {{/if}}
-          {{#if @job.latestDeployment.requiresPromotion}}
-            <button
-              data-test-promote-canary
-              type="button"
-              class="button is-warning is-small {{if this.promote.isRunning "is-loading"}}"
-              disabled={{this.promote.isRunning}}
-              onclick={{perform this.promote}}>Promote Canary</button>
-          {{/if}}
         </div>
       </div>
     </div>
-    <div class="boxed-section-body">
+    <div class="boxed-section-body {{if @job.latestDeployment.requiresPromotion "requires-promotion"}}">
+      {{#if @job.latestDeployment.requiresPromotion}}
+        {{#if this.canariesHealthy}}
+          <Hds::Alert @type="inline" @color="warning" as |A|>
+            <A.Title>Deployment requires promotion</A.Title>
+            <A.Description>Your deployment requires manual promotion — all canary allocations have passed their health checks.</A.Description>
+            <A.Button @text="Promote Canary" @color="primary" {{on "click" (perform this.promote)}} />
+          </Hds::Alert>
+        {{else}}
+          {{#if this.someCanariesHaveFailed}}
+            <Hds::Alert @type="inline" @color="critical" as |A|>
+              <A.Title>Some Canaries have failed</A.Title>
+              <A.Description>Your canary allocations have failed their health checks. Please have a look at the error logs and task events for the allocations in question.</A.Description>
+            </Hds::Alert>
+          {{else}}
+            <Hds::Alert @type="inline" @color="highlight" as |A|>
+              <A.Title>Checking Canary health</A.Title>
+              {{#if this.deploymentIsAutoPromoted}}
```

Comment:
deploymentIsAutoPromoted is a new property for the UI, and slightly changes our messaging during the "checking your allocs' health" phase.